docs/sphinx/developers/java-library.txt

```diff
@@ -1,39 +1,25 @@
 Using Bio-Formats as a Java library
 ===================================
 
-Bio-Formats as a Maven dependency
----------------------------------
-
-If you wish to make use of Bio-Formats within your own software it can be
-included as a dependency in any Maven project. The dependency can be added
-to the project pom file and should include the desired Bio-Formats version.
-Using `bioformats_package` as the artifactId will include the complete
-Bio-Formats package, or individual components can be chosen as desired.
-
-::
-
-  <dependency>
-    <groupId>ome</groupId>
-    <artifactId>bioformats_package</artifactId>
-    <version>5.3.0</version>
-  </dependency>
-
-In order to include this Bio-Formats dependency a custom repository
-must also be added to the project pom or
-:file:`${user.home}/.m2/settings.xml`.
-The repositories element is inherited so for a group of projects the
-repositories element can be defined at the top of your inheritance chain.
-
-::
-
-  <repositories>
-    <repository>
-      <id>ome</id>
-      <name>Bio-Formats Repo</name>
-      <url>http://artifacts.openmicroscopy.org/artifactory/maven</url>
-    </repository>
-  </repositories>
-  
+Bio-Formats as a Maven, Gradle or Ivy dependency
+------------------------------------------------
+
+All released :file:`.jar` artifacts are published to and can be obtained from
+the `OME Artifactory <http://artifacts.openmicroscopy.org/artifactory>`_.  The
+"Client Settings" section of the Artifactory main page provides example code
+snippets for inclusion into your Gradle, Maven or Ivy project, which will
+enable the use of this repository.
+
+Examples of getting started with Bio-Formats using Maven or Gradle are given
+in the https://github.com/openmicroscopy/bio-formats-examples repository.
+`OMERO <http://github.com/openmicroscopy/openmicroscopy>`_ uses Ivy to manage
+its Java dependencies including Bio-Formats.
+
+.. note::
+  In order to retrieve the NetCDF dependency of Bio-Formats, it is necessary
+  to configure the Unidata releases repository in addition to Maven Central and
+  the OME artifactory in your :file:`pom.xml`, :file:`build.gradle` or
+  :file:`ivy.xml` file.
```

Comment:
In the case of Maven and Gradle, https://github.com/openmicroscopy/bio-formats-examples/blob/master/pom.xml#L32 and https://github.com/openmicroscopy/bio-formats-examples/blob/master/build.gradle#L8 should include the declaration. I am a bit unsure about linking to a specific file line as nothing will break if there is a change but I could certainly link at the file URL. Thoughts?